test/stripe/stripe_object_test.rb

```diff
@@ -36,7 +36,10 @@ class StripeObjectTest < Test::Unit::TestCase
     end
 
     should "marshal a stripe object correctly" do
-      obj = Stripe::StripeObject.construct_from({ :id => 1, :name => 'Stripe' }, {:api_key => 'apikey'})
+      obj = Stripe::StripeObject.construct_from(
+        { :id => 1, :name => 'Stripe' },
+        { :api_key => 'apikey', :client => StripeClient.active_client }
```

Comment:
Would you mind just putting `client` here on a separate line and commenting that it's expected to be removed by the marshaling method because it can't be serialized?